pkg/framework/spi/crypto/api.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package crypto
+
+import "github.com/google/tink/go/keyset"
+
+// Provider interface for Crypto ctx
+type Provider interface {
+}
+
+// Creator method to create new pluggable Crypto service
+type Creator func(prov Provider) (Crypto, error)
+
+// Crypto is an interface defining a pluggable Crypto operations provider to Aries agents.
+// It's only responsible for pure crypto operations and nothing more. Envelope building and keys management are managed
+// in other interfaces
+type Crypto interface {
+	// Encrypt will encrypt msg and aad using a matching AEAD primitive in kh
```

Comment:
I'm bit confused with the wording "using a matching AEAD primitive". This is an interface definition - `"using"` word infers implementation. Shouldn't it be up-to the implementors to decide ?  If we want to force it to the implementation, then we `"must"` would be a better word. 

Same comment for other interface fn godocs.

Reply:
I agree - the wording should be removed from the godocs.